taichi/ir/frontend_ir.cpp

```diff
@@ -35,6 +35,13 @@ FrontendSNodeOpStmt::FrontendSNodeOpStmt(ASTBuilder *builder,
   }
 }
 
+FuncCallExpression::FuncCallExpression(ASTBuilder *builder,
+                                       Function *func,
+                                       const ExprGroup &args)
+    : func(func), args(args) {
+  this->args.exprs = builder->expand_expr(args.exprs);
```

Comment:
Why not do this in `ASTBuilder::insert_func_call_expr` and then pass the processed `ExprGroup` into the construction function?